src/test/java/com/artipie/rpm/RpmTest.java

```diff
@@ -165,11 +136,26 @@ public void addsSingleRpm() throws Exception {
         for (final String file : files) {
             MatcherAssert.assertThat(
                 Paths.get(
-                    home.toString(),
+                    store.toString(),
                     String.format("repodata/%s", file)
                 ).toFile().exists(),
                 Matchers.equalTo(true)
             );
         }
     }
+
+    private static Storage save(final Path folder, final Path store, final String key)
```

Comment:
@HDouss I 'dont like the idea of sharing this piece of code among all our test cases; a bug in here could get us false results from all our tests.

Reply:
@paulodamaso This part is asked by the puzzle. Besides, isn't that the benefit of behavior encapsulation/method extraction? If a bug is there, either a fix will fix all the tests, or the tests should be splitted because they don't require the same behavior anymore. Given that they share the same behavior for now, it is good to have that I think.

Reply:
@HDouss I prefer to have all code for creating the data needed in my test cases inside the test, but it is a particular preference. I consider that sharing a untested code (usually these helper function in tests aren't tested, since they are not production code) is more harmful than repeating very similar code for the creation of each test data. By the way, having to create lots of complex objects so we can run the tests may point to the fact that we have architectural problems in code, too.

Reply:
@paulidamaso I understand even I don't fully agree. What do we do with the fact that this is what asked by the puzzle/issue?

Reply:
@HDouss Don't change anything; I can fill a bug if I feel the need of refactoring it